backendless.d.ts

```diff
@@ -561,6 +563,7 @@ declare module Backendless {
      * @namespace Backendless.CustomServices
      **/
     let CustomServices: Backendless.BL.CustomServicesI;
+    let APIServices: Backendless.BL.CustomServicesI;
```

Comment:
it's already on the master branch, please update the branch

Reply:
Branch has been updated